prime/src/components/borrow/actions/UniswapAddLiquidityActionCard.tsx

```diff
@@ -62,6 +62,7 @@ export default function UniswapAddLiquidityActionCard(props: ActionCardProps) {
   // MARK: state for user inputs
   const [localIsAmount0UserDefined, setLocalIsAmount0UserDefined] = useState(false);
   const [localTokenAmounts, setLocalTokenAmounts] = useState<readonly [string, string]>(['', '']);
+  const [localLiquidity, setLocalLiquidity] = useState<JSBI>(JSBI.BigInt(0));
```

Comment:
We talked about this offline about a week and a half back. It fixes the issue where the user would change the token in an erroring add liquidity action. The error would disappear since it would default to zero liquidity (until updated) since we did not store the liquidity.